extensions/flutter-pub-dev-search/src/index.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  randomId,
+} from "@raycast/api";
+import { useState, useEffect, useRef } from "react";
+import fetch, { AbortError } from "node-fetch";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search for a Dart/Flutter package..." throttle>
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.id} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.name}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <CopyToClipboardAction
+              title="Copy Install Command"
+              content={`flutter pub add ${searchResult.name}`}
+            />
+          </ActionPanel.Section>
+          <ActionPanel.Section>
+            <OpenInBrowserAction 
+            title="Open in Browser" 
+            url={`https://pub.dev/packages/${searchResult.name}`}  
+            />
+          </ActionPanel.Section>
```

Comment:
```suggestion
          <CopyToClipboardAction
            title="Copy Install Command"
            content={`flutter pub add ${searchResult.name}`}
          />
          <OpenInBrowserAction 
            url={`https://pub.dev/packages/${searchResult.name}`}  
          />
```
- Title for OpenInBrowserAction in redundant since the default title is the same.
- Skip <ActionPanel.Section> so both commands have a shortcut to use.